src/test/java/no/unit/nva/model/instancetypes/DegreeBachelorTest.java

```diff
@@ -0,0 +1,144 @@
+    @ParameterizedTest
+    @CsvSource({
+            "i,xxviii,398,true,true,true,",
+            ",,231,false,true,true",
+            ",,123,true,false,false"
+    })
+    void objectMapperReturnsDegreeBachelorWhenInputIsValidString(String begin,
+                                                                 String end,
+                                                                 String pages,
+                                                                 boolean illustrated,
+                                                                 boolean peerReviewed,
+                                                                 boolean openAccess
+    )
+            throws JsonProcessingException, InvalidPageRangeException, InvalidPageTypeException {
+        DegreeBachelor expected = generateDegreeBachelor(
+                begin,
+                end,
+                pages,
+                illustrated,
+                peerReviewed,
+                openAccess
+        );
+        String json = generateBookInstanceJson(
+                DEGREE_BACHELOR,
+                begin,
+                end,
+                pages,
+                illustrated,
+                peerReviewed,
+                openAccess);
+        DegreeBachelor actual = objectMapper.readValue(json, DegreeBachelor.class);
+        assertEquals(expected, actual);
+    }
+
+    @DisplayName("DegreeBachelor: ObjectMapper serializes valid input correctly")
+    @ParameterizedTest
+    @CsvSource({
+            "i,xxviii,398,true,true,true,",
+            ",,231,false,true,true",
+            ",,123,true,false,false"
+    })
+    void objectMapperReturnsExpectedJsonWhenInputIsValid(String begin,
+                                                         String end,
+                                                         String pages,
+                                                         boolean illustrated,
+                                                         boolean peerReviewed,
+                                                         boolean openAccess) throws InvalidPageTypeException,
+            JsonProcessingException, InvalidPageRangeException {
+        DegreeBachelor degreeBachelor = generateDegreeBachelor(
+                begin,
+                end,
+                pages,
+                illustrated,
+                peerReviewed,
+                openAccess
+        );
+        String json = objectMapper.writeValueAsString(degreeBachelor);
+        String expected = generateBookInstanceJson(
+                DEGREE_BACHELOR,
+                begin,
+                end,
+                pages,
+                illustrated,
+                peerReviewed,
+                openAccess);
+        assertEquals(expected, json);
+    }
+
+    @DisplayName("DegreeBachelor throws InvalidPageTypeException if pages is not MonographPages")
+    @Test
+    void degreeBachelorThrowsInvalidPageTypeExceptionWhenInputIsRange() {
```

Comment:
WhenInputIsRange -> WhenInputIsNotMonographPages ? Although you do provide this information in the display message